fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProvider.java

```diff
@@ -587,6 +591,7 @@ private Query getFilterQuery(Set<ApiFilter> filters, int perPage) {
             throw new RuntimeException(e);
```

Comment:
I still don't see a timing differentiation between talking to Lucene and fetching the dimension rows using the Lucene results. Did you want to break those up?